resources/gcp/gke/cluster.go

```diff
@@ -7,14 +7,15 @@ import (
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 )
 
-func NewCluster(e gcp.Environment, name string, opts ...pulumi.ResourceOption) (*container.Cluster, pulumi.StringOutput, error) {
+func NewCluster(e gcp.Environment, name string, autopilot bool, opts ...pulumi.ResourceOption) (*container.Cluster, pulumi.StringOutput, error) {
 	opts = append(opts, e.WithProviders(config.ProviderGCP))
 
 	cluster, err := container.NewCluster(e.Ctx(), e.Namer.ResourceName(name), &container.ClusterArgs{
-		InitialNodeCount:   pulumi.Int(2),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Is it expected to remove `InitialNodeCount`?

Reply:
Nice catch. Actually it breaks autopilot if we set it to 2. But it break classic GKE if we do not set it all. Updating the value to 1 which should be ok for both the deployments